install/prepare_compute_nodes.md

```diff
@@ -14,33 +20,41 @@ See [Update the Gigabyte Server BIOS Time](../operations/node_management/Update_
 ## Details
 
 <a name="configure-hpe-apollo-6500-x645d-gen10-plus-compute-nodes"></a>
+
 ### 1. Configure HPE Apollo 6500 XL645d Gen10 Plus Compute Nodes
 
-The HPE Apollo 6500 XL645d Gen10 Plus compute node uses a NIC/shared iLO network port. The NIC is
-also referred to as the Embedded LOM (LAN On Motherboard) and is available to the booted OS. This
-shared port is plugged into a port on the TOR Ethernet switch designated for the
-Hardware Management Network (HMN) causing the NIC to get an IP address assigned
-to it from the wrong pool. To prevent this from happening, the iLO VLAN tag
-needs to be configured for VLAN 4 and the switch port the NIC/shared iLO is
-plugged into needs to be configured to allow only VLAN 4 traffic. This prevents
-the NIC from communicating over the switch, and it will no longer DHCP an IP
-address.
+The HPE Apollo 6500 XL645d Gen10 Plus compute node uses a NIC/shared iLO network
+port. The NIC is also referred to as the Embedded LOM (LAN On Motherboard) and
+is available to the booted OS. This shared port is plugged into a port on the
+TOR Ethernet switch designated for the Hardware Management Network (HMN) causing
+the NIC to get an IP address assigned to it from the wrong pool. To prevent this
+from happening, the iLO VLAN tag needs to be configured for VLAN 4 and the
+switch port the NIC/shared iLO is plugged into needs to be configured to allow
+only VLAN 4 traffic. This prevents the NIC from communicating over the switch
+and it will no longer DHCP an IP address.
+
+This procedure needs to be done for each of the HPE Apollo 6500 XL645d servers
+that will be managed by CSM software. River compute nodes always index their
+BMCs from 1. For example, the compute BMCs in servers with more than one node
+will have xnames as follows x3000c0s30b1, x3000c0s30b2, x3000c0s30b3, etc...
+The node indicator is always a 0. For example, x3000c0s30b1n0 or x3000c0s30b4n0.
 
-This procedure needs to be done for each HPE Apollo 6500 XL645d node managed by CSM software.
+   <a name="gather_information"></a>
 
 1. Gather Information
 
-   Example using x3000c0s30b0n0 as the target component xname
+   Example using x3000c0s30b1n0 as the target compute node xname
```

Comment:
Change to:

The following is an example using x3000c0s30b1n0 as the target compute node xname: